enterprise-beans/4.0/_index.md

```diff
@@ -15,10 +15,17 @@ Jakarta Enterprise Beans defines an architecture for the development and deploym
 * Maven coordinates
   * [jakarta.ejb:jakarta.ejb-api:jar:4.0.0]()
 
+The Jakarta Enterprise Beans 4.0 intended scope includes all plans detailed in the [Jakarta EE 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan), which in essence involves a backwards incompatible namespace change from `javax.ejb` to `jakarta.ejb`.  In addition the Jakarta Enterprise Beans 4.0 Release Plan will involve removal of `java.security.Identity`, which is being removed from future versions of Java.
```

Comment:
Further up on line 9, you should include the url for the EJB 4.0 Release Record, which you will be creating...  :-)